Student/StudentUnitTests/Helpers/TestEnvironment.swift

```diff
@@ -20,13 +20,47 @@ import Foundation
 @testable import Core
 import TestsFoundation
 import CoreData
+import XCTest
 
-func testEnvironment() -> AppEnvironment {
-    let env = AppEnvironment.shared
-    env.api = URLSessionAPI(loginSession: .make(), urlSession: MockURLSession())
-    env.database = singleSharedTestDatabase
-    env.globalDatabase = singleSharedTestDatabase
-    env.router = TestRouter()
-    env.logger = TestLogger()
-    return env
+//func testEnvironment() -> AppEnvironment {
+//    let env = AppEnvironment.shared
+//    env.api = URLSessionAPI(loginSession: .make(), urlSession: MockURLSession())
+//    env.database = singleSharedTestDatabase
+//    env.globalDatabase = singleSharedTestDatabase
+//    env.router = TestRouter()
+//    env.logger = TestLogger()
+//
+//    return env
+//}
```

Comment:
Do you want to preserve this? Or can we remove it?

Reply:
If we are good with the new `TestEnvironment` and `TestStore` I am ok removing this.

Reply:
Good with me.